libcaf_test/caf/test/dsl.hpp

```diff
@@ -632,12 +632,12 @@ class test_coordinator_fixture {
   // -- constructors, destructors, and assignment operators --------------------
 
   static Config& init_config(Config& cfg) {
+    cfg.set("logger.file-verbosity", caf::atom("quiet"));
```

Comment:
Fixing #959 as a side effect here?

Reply:
Apparently yes, I needed a way to look at the log output while working on this.